ios/Rainbow.xcodeproj/project.pbxproj

```diff
@@ -733,17 +747,17 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 13B07F931A680F5B00A75B9A /* Build configuration list for PBXNativeTarget "Rainbow" */;
 			buildPhases = (
-				A1E5EC36516AA3B0EE46BE1F /* [CP] Check Pods Manifest.lock */,
+				09027C46E77CF77316A39F45 /* [CP] Check Pods Manifest.lock */,
+				5E71112ED85590D3BF2BCC85 /* [Expo] Configure project */,
 				13B07F871A680F5B00A75B9A /* Sources */,
 				13B07F8C1A680F5B00A75B9A /* Frameworks */,
 				13B07F8E1A680F5B00A75B9A /* Resources */,
 				00DD1BFF1BD5951E006B06BC /* Bundle React Native code and images */,
-				9FF961FEA7AF435FA18ED988 /* Upload Debug Symbols to Sentry */,
```

Comment:
removing this temporarily, will follow up with [app 960](https://linear.app/rainbow/issue/APP-960/ensure-sentry-is-configured-and-working)